### PR TITLE
Find command notation

### PR DESCRIPTION
**Pull request by ongweijie7**
Edit syntax of Find Command notation to match Add Command

### CODE REVIEW

**Comment by ongweijie7** [comment]
Edited Find command syntax to be aligned with Add Command and also removed unnecessary command which is similar to 'list'. Test cases have also been edited.

**Review by CedricPei — APPROVED** [review]
LGTM

**Review by SeanChinJunKai — APPROVED** [review]
LGTM.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -91,23 +91,21 @@ Examples: `delete i/T0012345A` to delete patient with NRIC T0012345A from patien
...
 ### Find persons by name or address: `find`
 
-Find persons according to a particular attribute. Eg. name(/n), address(/a), nric(/nric)
-Returns unfiltered person list if `/all` is passed as an attribute
-Format: `find /ATTRIBUTE [KEYWORD] [MORE_KEYWORDS]`
-
+Find persons according to a particular attribute stated followed by the change. Eg. name(n/), address(a/), nric(i/)
+Format: `find attribute/keyword [MORE_KEYWORDS]`
 
+* Only the provided attribute will be search
```
Comment (Creationsv2) [inline docs/UserGuide.md:97]: Slight grammar error, perhaps you meant:

The search will be carried out only for the provided attribute

File: docs/UserGuide.md [hunk 2]
```diff
@@ -91,23 +91,21 @@ Examples: `delete i/T0012345A` to delete patient with NRIC T0012345A from patien
...
+* Only attribute can be searched at one time
```
Comment (Creationsv2) [inline docs/UserGuide.md:98]: Slight grammar error, perhaps you meant:

Only one attribute can be searched for at one time

Comment (ongweijie7) [reply docs/UserGuide.md]: changes made

**Review by Creationsv2 — APPROVED** [review]
LGTM. Good job.

**Pull request merged**